cli/tenzir_platform/helpers/oidc.py

```diff
@@ -153,7 +153,7 @@ def _device_code_flow(self) -> dict[str, str]:
             )
             token_data = token_response.json()
             if token_response.status_code == 200:
-                print("Authenticated!", file=sys.stderr)
+                print("Authenticated!")
```

Comment:
Why does this now go to stdout? Doesn't this break certain shell pipe use cases?
How do you decide what goes to stdout vs stderr?